src/octoprint/util/gcodeInterpreter.py

```diff
@@ -484,6 +494,15 @@ def _load(
                 else:
                     e = 0
 
+                # If move, calculate new min/max coordinates
+                if move:
+                    # store as print move if extrusion is > 0
+                    if e > 0:
```

Comment:
This shouldn't be a problem, if `e` is None it will be set to `0` right before this.

```python
if e is not None:
    ...
else:
    e = 0
```

Reply:
Ah, thank you, I managed to miss that